corehq/apps/reports/util.py

```diff
@@ -236,21 +239,25 @@ def parts():
             if full_name:
                 yield ' "%s"' % html.escape(full_name)
         username_in_report = safestring.mark_safe(''.join(parts()))
-        return SimplifiedUserInfo(
+        info = SimplifiedUserInfo(
             user_id=user.get('_id', ''),
             username_in_report=username_in_report,
             raw_username=raw_username,
-            is_active=user.get('is_active', None)
+            is_active=user.get('is_active', None),
+            location_id=user.get('location_id', None)
         )
+        if '__group_ids' in user:
```

Comment:
This block could probably go in the `__init__` function as well

Reply:
the `SimplifiedUserInfo` extends a namedtuple so making changes to `__init__` is a bit more complicated.